docs/rancher.md

```diff
@@ -0,0 +1,29 @@
+# Deploying Antrea on Rancher Managed Cluster
+
+## Deploy Rancher Server UI Dashboard
+
+Follow these [steps](https://rancher.com/docs/rancher/v1.6/en/installing-rancher/installing-server/) to deploy rancher server ui dashboard.
+
+## Deploy Antrea Using Rancher UI
+
+* Sign-in to the UI dashboard, and switch to cluster management.
+
+<img src="https://downloads.antrea.io/static/04062023/rancher_cluster_management.png" width="600">
+
+* Create new Node template there under the RKE configuration section.
+
+<img src="https://downloads.antrea.io/static/04062023/rancher_node_template.png" width="600">
+
+* After saving the template, proceed with creating new cluster, under the clusters tab click on create button, then enter the name, and number of worker and controlplane Nodes respectively.
+
+<img src="https://downloads.antrea.io/static/04062023/rancher_create.png" width="600">
```

Comment:
I feel the width can be 800 or 1000, please check and make it clearer with default width.

Reply:
okay i will make it 800, but in some docs the width is 600 as well.